### ISSUE TITLE
E0308 libc false positive errors, expected *const i8, found *const u8

### ISSUE THREAD

**Issue by dvic** [intellij-rust/intellij-rust#2760]
## Environment

* **Intellij-Rust plugin version:**
`0.2.0.2103-182`
* **Rust toolchain version:**
```
   stable-x86_64-apple-darwin unchanged - rustc 1.28.0 (9634041f0 2018-07-30)
     beta-x86_64-apple-darwin unchanged - rustc 1.29.0-beta.2 (ea82e08ee 2018-08-05)
  nightly-x86_64-apple-darwin unchanged - rustc 1.30.0-nightly (73c78734b 2018-08-05)
```
* **IDE name and version:**
`CLion 2018.2` (but problem also occurs in Intellij IDEA)

* **Operating system:**
`MacOS X Sierra`

## Problem description
The argument to `bar` in the below examples is highlighted with error ``expected `*const i8`, found `*const u8` ``, while the code compiles correctly on my machine.

## Steps to reproduce
```rust
use libc::c_char;
use std::ffi::{CStr, CString};

pub fn foo() {
    let c_input = CString::new("").unwrap();
    bar(c_input.as_ptr())
}

fn bar(_: *const c_char) {}
```

**Comment by ortem** [comment]
Fixed in https://github.com/intellij-rust/intellij-rust/pull/4232.
Follow https://github.com/intellij-rust/intellij-rust/issues/1191 to track the progress of `cfg` support.
